fab/fab/operations/supervisor.py

```diff
@@ -107,7 +107,8 @@ def set_celery_supervisorconf():
             conf_destination_filename = 'supervisor_celery_worker_{}_{}.conf'.format(
                 comma_separated_queue_names, worker_num)
 
-            worker_name = get_celery_worker_name(params['comma_separated_queue_names'],
+            worker_name = get_celery_worker_name(env.env_name,
```

Comment:
@mkangia I just noticed this: `env.env_name` isn't the right variable here, even though most of the time it'll have the same value. What you need is `environment.meta_yml.deploy_env`, which I believe in fabric is already set on the fabric `env` variable as `env.deploy_env`.